### PR TITLE
style: apply Google-java-format to persistentworkers

### PR DESCRIPTION
**Pull request by jasonschroeder-sfdc**
the `persistentworkers` folder wasn't getting formatted. Add it to the rule and format it.

### CODE REVIEW

File: persistentworkers/src/main/java/persistent/bazel/client/WorkerKey.java [hunk 1]
```diff
@@ -144,26 +123,28 @@ public int hashCode() {
...
+        + args
+        + ",\n\t"
+        + "env="
+        + env
+        + ",\n\t"
+        + "mnemonic="
+        + mnemonic
+        + ",\n\t"
+        + "execRoot="
+        + execRoot.toAbsolutePath()
```
Comment (werkt) [inline persistentworkers/src/main/java/persistent/bazel/client/WorkerKey.java:147]: oof, some noformat here maybe?

Comment (jasonschroeder-sfdc) [reply persistentworkers/src/main/java/persistent/bazel/client/WorkerKey.java:147]: There's no `noformat` for google-java-format :dead:
https://github.com/google/google-java-format/wiki/FAQ#why-cant-i-use-magic-comments-to-disable-formatting-for-certain-sections-of-code

I can convert it to a Lombok `@ToString`?

Comment (werkt) [reply persistentworkers/src/main/java/persistent/bazel/client/WorkerKey.java:147]: Yeah, that looks like a better idea

**Review by 80degreeswest — APPROVED** [review]
🧹

**Pull request merged**